Detectors/MUON/MCH/Clustering/include/MCHClustering/ClusterFinderGEM.h

```diff
@@ -136,7 +133,11 @@ class ClusterFinderGEM
   void setClusterResolution(Cluster& cluster) const;
   std::unique_ptr<MathiesonOriginal[]> mMathiesons; ///< Mathieson functions for station 1 and the others
   // GG MathiesonOriginal* mMathieson = nullptr;          ///< pointer to the Mathieson function currently used
-
+  //
+  // GG Introduced for run3
+  // function to reinterpret digit ADC as charge
+  std::function<double(uint32_t)> mADCToCharge = [](uint32_t adc) { return static_cast<double>(adc); };
```

Comment:
seems to be a rather complicated way of doing a simple cast. Or is it placeholder for a foreseen more complex ADC to charge conversion ?

Reply:
I agree.
If I remember, I used the cast function in old implementation.
I don't really focus on it.
Thanks